tools/train.py

```diff
@@ -35,15 +35,10 @@ def parse_args():
         help='whether not to evaluate the checkpoint during training')
     group_gpus = parser.add_mutually_exclusive_group()
     group_gpus.add_argument(
-        '--gpus',
+        '--gpu-id',
         type=int,
-        help='number of gpus to use '
```

Comment:
We should add deprecation warning of --gpu-ids and --gpus to indicate users that they are deprecated and useless. We should not delete them immediately.

Reply:
ok